File: src/entities/enemies/FastEnemy.ts
import { Enemy } from './Enemy';


export class FastEnemy extends Enemy {
    protected initStats(): void {
        this.maxHealth = 100 + 100 * this.healthLevel;
        this.health = this.maxHealth;
        this.maxSpeed = 120;
        this.speed = this.maxSpeed;
    }

    getSpriteName(): string {
        return 'scout';
    }

}

File: original-java-game/src/enemies/FastEnemy.java
package enemies;

import source.Board;

public class FastEnemy extends Enemy {

	public FastEnemy(Board gui, int xs, int ys, int nivå) {
		super(gui.getBilder().getScout(), gui, xs, ys, gui.getLevel(), nivå);
	}

	@Override
	protected void setStats() {
		maxhealth = 100 + 100 * healthlevel;
		maxspeed = 1.7;
		damage = 1;
	}

}//END
